specs/data_structures.md

```diff
@@ -76,7 +76,7 @@ Data Structures
 | `Amount`                    | `uint64`                   |
 | [`BlockID`](#blockid)       | [HashDigest](#hashdigest)  |
 | `FeeRate`                   | `uint64`                   |
-| `Graffiti`                  | `NamespaceID`              |
+| `Graffiti`                  | `byte[GRAFFITI_BYTES]`     |
```

Comment:
Why did this use to be a `NamespaceID` and now is bytes? Because now the Graffiti and namespaces differ in size?

Reply:
Because we don't necessarily want to change the size of graffiti. It being big enough to put an arbitrary hash or some other data is important I think.

Reply:
That makes sense! I guess I was just wondering why it previously was typed as a `NamespaceID` then.

Reply:
Same